services/api/index.ts

```diff
@@ -0,0 +1,104 @@
+import type { UseQueryOptions } from '@tanstack/react-query';
+import { useQueries } from '@tanstack/react-query';
+import { EVENT_SLUG, ORG_SLUG } from '../../config/constants';
+import type { IOrganizers, ISessions, ISpeaker, ISponsors } from '../../global';
+import { axiosInstance } from './axios';
+import { queryClient } from './react-query';
+
+export const getSessions = async (count: number) => {
+  const { data } = await axiosInstance.get(`/events/${EVENT_SLUG}/sessions`, { params: { per_page: count } });
```

Comment:
We should add type information to all these responses we receive to prevent `any`'s from leaking into the codebase.
e.g. `axiosInstance.get<ISessions>(`/events/`.